hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroUtils.java

```diff
@@ -704,10 +705,25 @@ public static Schema getNullSchema() {
    * @return sanitized name
    */
   public static String sanitizeName(String name) {
-    if (name.substring(0, 1).matches(INVALID_AVRO_FIRST_CHAR_IN_NAMES)) {
-      name = name.replaceFirst(INVALID_AVRO_FIRST_CHAR_IN_NAMES, MASK_FOR_INVALID_CHARS_IN_NAMES);
+    return sanitizeName(name, MASK_FOR_INVALID_CHARS_IN_NAMES);
```

Comment:
where is this method used. should we remove this. or use the default value for config AVRO_FIELD_NAME_INVALID_CHAR_MASK

Reply:
This is used in `HoodieSparkBootstrapSchemaProvider` and `AvroConversionUtils.Scala`. Can't remove this. `AVRO_FIELD_NAME_INVALID_CHAR_MASK` is scoped to deltastreamer source as we tackle this santization in limited cases only. So I believe uniification can be done when the sanitization is implemented for all cases